City-of-Pori/pori-kada
# PORI-282 Updated WYSIWYG settings to include new image size, styled i…

…mages inside content area, fixed feature conflicts

To test:

1. drush fra -y && drush cc all
2. Go to edit any page and drag and drop an image from scald to the body text area.
3. Right click the image and choose "Edit atom properties".
4. Remove the checkbox from "Add caption" and make sure you have the following options available in the "Context": Content Full, Content Half, Content 250 wide.
5. Choose Content 250 wide and align it to the right.
6. Save and make sure the text goes around the image on desktop.

[sisaltoalueen_kuvan_ohjeet.pdf](https://github.com/City-of-Pori/pori-kada/files/1706668/sisaltoalueen_kuvan_ohjeet.pdf)
